Dockerfile.app

```diff
@@ -40,7 +40,6 @@ WORKDIR /app/examples/vic-gov-au/
 # force it to load the environment variable during build time. Otherwise it cannot read $LAGOON_GIT_BRANCH.
 RUN  . /home/.bashrc \
     && yarn run build --modern=client \
-    && chmod -R 755 ~/.config \
```

Comment:
Lagoon failed building as this throws "not found" error. Removing this seems to completes the build fine. 
No build issue in CI.